django/core/jinja2/core/jobs/retrieve.jinja

```diff
@@ -16,6 +16,22 @@
     ]) }}
 {% endblock %}
 
+{% block ogp_tags %}
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="{{ request.build_absolute_uri(absolute_url) }}">
+    <meta property="og:type" content="article">
+    <meta property="og:title" content="{{ title }}">
+    <meta property="og:description" content="{{ description }}">
```

Comment:
probably best to use `summary` instead, as description can be quite long

same thing with Events